operations/patch_util.go

```diff
@@ -271,8 +271,15 @@ func (p *patchParams) validatePatchCommand(ctx context.Context, conf *ClientSett
 		}
 	}
 
-	if (len(p.Tasks) == 0 || len(p.Variants) == 0) && p.Alias == "" && p.Finalize {
-		return ref, errors.Errorf("Need to specify at least one task/variant or alias when finalizing.")
+	if !p.Finalize || p.Alias != "" {
+		return ref, nil
```

Comment:
instead of return here, you could just move the if statements below into the block so we dont have to return the same thing on 278

Reply:
Fixed!